src/solvers/dgsem_tree/dg_2d_parabolic.jl

```diff
@@ -589,7 +820,19 @@ function calc_gradient!(gradients, u_transformed, t,
                                       dg.surface_integral, dg)
     end
 
-    # TODO: parabolic; mortars
+    # Prolong solution to mortars
+    @trixi_timeit timer() "prolong2mortars" begin
+        prolong2mortars!(cache, u_transformed, mesh, equations_parabolic,
+                         dg.mortar, dg.surface_integral, dg)
+    end
+
+    # Calculate mortar fluxes
+    @trixi_timeit timer() "mortar flux" begin
+        calc_mortar_flux!(surface_flux_values,
+                          mesh,
+                          equations_parabolic,
+                          dg.mortar, dg.surface_integral, dg, cache)
+    end
```

Comment:
Good catch, the reason I resort for `prolong2mortars to the hyperbolic implementation is due to the fact that the `mortar` datastructure lies only in the (hyperbolic) `cache`, which makes this re-use very attractive.

`calc_mortar_flux`, however, has its own implementation to avoid the need to pass information on `nonconservative_terms`, i.e., the implementation defaults to conserved terms only, as  `calc_interface_flux!` for parabolic terms.